style.css

```diff
@@ -0,0 +1,101 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+nav {
+  display: flex;
+  flex-direction: column;
+  position: absolute;
+  background-color: #3c3a39;
+  height: 70px;
+  width: 375px;
+  color: #fff;
+  justify-content: center;
+}
+
+ul {
+  display: flex;
+  justify-content: space-between;
+  list-style-type: none;
+  padding: 0 20px 0 20px;
+}
+
+li {
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-size: 18px;
+  font-style: normal;
+  font-weight: 700;
+  line-height: 24px;
+}
+
+a {
+  text-decoration: none;
+  color: #fff;
+}
+
+.main-section {
+  background-color: #1c1a19;
+  width: 375px;
+  height: 670px;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  padding-top: 9rem;
+  padding-left: 2.5rem;
+  padding-right: 2.5rem;
+}
+
+.headline,
+p {
+  color: #ff6b00;
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+}
+
+.headline,
+h4 {
+  color: #fff5e1;
+  font-size: 24px;
+  font-weight: bold;
+  line-height: 32px;
+  font-style: normal;
+}
+
+.headline,
+h6 {
+  color: #fff;
+  font-size: 16px;
+  line-height: 24px;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+}
```

Comment:
- [ ] Kindly use the correct font-family for each element used in the figma design as it is required to follow the design

Reply:
Yes, I agree. I have done that. Thank you.